examples/asr/asr_cache_aware_streaming/speech_to_text_cache_aware_streaming_infer.py

```diff
@@ -259,6 +264,13 @@ def main():
         help="Enables padding the audio input and then dropping the extra steps after the pre-encoding for the first step. It makes the outputs of the downsampling exactly as the offline mode for some techniques like striding.",
     )
 
+    parser.add_argument(
+        "--set_decoder",
+        choices=["ctc", "rnnt"],
```

Comment:
why not give 'rnnt+ctc' option to decoder as well?

Reply:
That option is not feasible as just one decoder can be used for inference time. To use both, the whole inference process needs to be done twice with different decoders.